scripts/validate/yaml-lint

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env php
+
+<?php
+
+$vendor_dir = getenv('COMPOSER_VENDOR_DIR');
+
+if (empty($vendor_dir)) {
```

Comment:
Added autoloader detection to try and reduce the strict coupling with the docker file structure.